src/components/Pop/Arrow.js

```diff
@@ -123,4 +123,16 @@ export const getPosition = (placement: string) => {
   return placement.split('-')[1]
 }
 
+// This is primarily for popper.js within the test JSDOM environment
+// https://github.com/FezVrasta/popper.js/issues/478
+export const sanitizeStyles = style => {
+  const { left, top } = style
+
+  return {
+    ...style,
+    left: isNaN(left) ? 0 : left,
+    top: isNaN(top) ? 0 : top,
+  }
```

Comment:
We should maybe make a util function for this. Like `safeParseInt` that returns a zero for NaN values.

Reply:
@knicklabs OoOOooO! I like that!